src/modules/todolist.js

```diff
@@ -0,0 +1,72 @@
+export default class TodoList {
+  constructor() {
+    this.tasks = [];
+  }
+
+  addTask = (description) => {
+    const completed = false;
+    const id = this.tasks.length + 1;
+    const task = {
+      id,
+      description,
+      completed,
+    };
+    this.tasks.push(task);
+  }
+
+  removeList = (id) => {
+    this.tasks.forEach((task) => {
+      if (task.id === id) {
+        this.tasks.splice(this.tasks.indexOf(task), 1);
+      }
+    });
+  }
+
+  displayToDo(list) {
+    list.innerHTML = '';
+    this.tasks.forEach((task) => {
+      const li = `<div id="${task.id}" class="list">
+      <input
+        type="checkbox"
+        id="${task.id}"
+        name="task"
+        value="task"
+        ${task.completed ? 'checked' : ''}
+        class="checkbox"
```

Comment:
- [ ] When a user checks the completed `checkbox`, the status should change from `false` to `true`, and clicking the `clear all completed`  button should delete all checked tasks. Kindly implement this as it is a requirement

Reply:
Thank you @Munsa1  for your comment, kindly check again for today's project requirement, because I don't see where it's required to  Implement a function for the "Clear all completed" button.
I think this one should be for the next project.